lib/interfaces/uncaught.js

```diff
@@ -54,6 +54,7 @@ function handlerSetup(client, config) {
 
     errorHandlerRouter(err, em);
     client.sendError(em, handleProcessExit);
+    setTimeout(handleProcessExit, 2000);
```

Comment:
Sorry, publish may have been a misleading word. I want to make sure that we have a test that fails if we cease to report errors if an unhandled exception occurs. Here is what we do in the trace agent: https://github.com/GoogleCloudPlatform/cloud-trace-nodejs/blob/master/test/standalone/test-trace-uncaught-exception.js. There are some timers in play but we have not had flakiness troubles with this test on the trace end.